app/views/case_logs/index.html.erb

```diff
@@ -1,14 +1,14 @@
-<% content_for :title, "Logs" %>
+<% title = @pagy.pages > 1 ? "Logs (page #{@pagy.page} of #{@pagy.pages})" : "Logs" %>
```

Comment:
Probably fine for this PR, but DRYer if the title for any page using `@pagy` shows the page number and number of pages. Also ensures this accessibility feature isn’t missed on subsequent paged views.

Reply:
Done